eo.json

```diff
@@ -1338,6 +1343,9 @@
         "Tutorial.AvatarTutorial.EquippingAndFavoriting.Description" : "Post kiam vi havos avataron, ekportu ĝin klakante sur ĝi kaj elektante Porti Avataron.<br><br>Konservu vian nun portatan avataron alklakante <sprite name=Download>  .<br><br>Vi povas preferatigi vian avataron elektante ĝin en via<br>inventaro kaj alklakante <sprite name=Favorite>  .",
         "Tutorial.AvatarTutorial.AvatarCreation.Description" : "Por importi kaj agordi propran avataron, iru al<br>la Lernilo pri Kreado de Neos-avataroj",
 
+        "Tutorial.Intro.PublicFolders.Title" : "Publikaj Dosierujoj",
+        "Tutorial.Intro.PublicFolders.Description" : "Publikaj Dosieruoj estas facila vojo por uzantoj en la Komunumo konigi kaj ĝisdatigi iliajn Faraĵojn kiel Avataroj.<br><br>Por konservi, ekprenu la dosierujon, kaj dum tenado premu <sprite name=Add>  en via Inventaro.<br><br>Vi ankaŭ povas konservi per via Kunteksta Menuo.<br><br>Provu per <b>The Starter Pack</b>. Dosierujo desegnita por novaj uzantoj.",
```

Comment:
The last phrase should be "Dosierujo _dezajnita_ [...]" instead of "desegnita"

Reply:
Dankon! I have made the correction.